sidebarGeneral.ts

```diff
@@ -66,7 +66,7 @@ export const sidebarGeneral: SidebarsConfig = {
             'general/manage-tokens/holding-rose-tokens/ledger-wallet',
           ]
         },
-        'general/manage-tokens/how-to-transfer-eth-erc20-to-emerald-paratime',
+        'general/manage-tokens/how-to-transfer-rose-erc20',
```

Comment:
Let's keep this chapter focused on bridging.

```suggestion
        'general/manage-tokens/how-to-bridge-assets',
```

Reply:
Also, don't forget to set up redirection from the previous URL.